packages/docs-app/src/examples/select-examples/films.tsx

```diff
@@ -145,6 +146,41 @@ export const filterFilm: ItemPredicate<IFilm> = (query, film) => {
     return `${film.rank}. ${film.title.toLowerCase()} ${film.year}`.indexOf(query.toLowerCase()) >= 0;
 };
 
+function highlightText(text: string, query: string) {
+    let lastIndex = 0;
+    const words = query
+        .split(/\s+/)
+        .filter(word => word.length > 0)
+        .map(regexEscape);
+    if (words.length === 0) {
+        return [text];
+    }
+    const regexp = new RegExp(words.join("|"), "gi");
+    const tokens: React.ReactNode[] = [];
+    while (true) {
+        const match = regexp.exec(text);
+        if (!match) {
+            break;
+        }
+        const length = match[0].length;
+        const before = text.slice(lastIndex, regexp.lastIndex - length);
+        if (before.length > 0) {
+            tokens.push(before);
+        }
+        tokens.push(<span style={{ fontWeight: "bold" }}>{match[0]}</span>);
```

Comment:
use `<strong>` tag instead please

Reply:
Good call. I completely forgot that `<strong>` was a thing. 🤦